drizzle/0000_flowery_union_jack.sql

```diff
@@ -0,0 +1,43 @@
+CREATE TABLE IF NOT EXISTS "code_group_mappings" (
+	"id" serial PRIMARY KEY NOT NULL,
```

Comment:
do we get to control the name of this file so that we can use something more meaningful?

Reply:
We can set custom migration file names by providing --name CLI option like
`npx drizzle-kit generate --name=init`

Reply:
I think the current name could be very confusing if someone doesn't understand drizzle.